packages/chatbot-server-mongodb-public/evalCases/custom_tool_conversations.yml

```diff
@@ -0,0 +1,182 @@
+- name: custom_tool_no_parameters
+  messages:
+    - role: user
+      content: What MongoDB courses are available? 
+  expectation: A 'mongodb_university_lookup' is returned
+  expectedMessageDetail:
+    - role: user
+    - role: assistant-tool
+      toolCallName: mongodb_university_lookup
+  customTools:
+    - name: mongodb_university_lookup
+      description: Fetches the current catalog of MongoDB University courses. 
+  tags:
+    - customTools
+- name: custom_tool_with_parameters
+  messages:
+    - role: user
+      content: >
+        Can you help debug my code? It's not doing anything
+        
+        def bubble_sort(arr):
+          n = len(arr)
+          for i in range(n):
+              for j in range(0, n - i):
+                  if arr[j] > arr[j + 1]:
+                      arr[j], arr[j + 1] = arr[j + 1], arr[j]
+  expectation: A 'code_execution' tool call is returned
+  expectedMessageDetail:
+    - role: user
+    - role: assistant-tool
+      toolCallName: code_execution
+  customTools:
+    - name: code_execution
+      description: Executes a given code snippet and returns the output. This tool should be called when debugging  or troubleshooting to get code execution results
+      parameters:
+        code:
+          type: string
+          description: The code snippet to execute.
+        language:
+          type: string
+          description: The programming language of the code snippet.
```

Comment:
nit: i'd reverse the order on these w/ `language` before `code` b/c the model will generate that first